marklogic-data-hub/src/main/java/com/marklogic/hub/impl/ScaffoldingImpl.java

```diff
@@ -128,25 +128,21 @@ public static String getAbsolutePath(String first, String... more) {
         }
 
         if (stepDefName != null && stepDefinitionManager.getStepDefinition(stepDefName, StepDefinitionType.getStepDefinitionType(stepType)) == null) {
-            stepDefinition = StepDefinition.create(stepDefName, StepDefinitionType.getStepDefinitionType(stepType));
+            try{
```

Comment:
No need to change now, but these are great candidates for separate private methods, particularly to make this fairly long method more readable.